cmd/unifi_exporter/main.go

```diff
@@ -9,49 +9,69 @@ import (
 	"net/http"
 	"strings"
 	"time"
-
+	"strconv"
+	"io/ioutil"
```

Comment:
Please run goimports on this file to tidy imports up and also format it: https://godoc.org/golang.org/x/tools/cmd/goimports.